src/main/java/edu/cornell/kfs/fp/document/validation/impl/CuDisbursementVoucherPayeeStateAndCountryValidation.java

```diff
@@ -50,5 +65,16 @@ public AccountingDocument getAccountingDocumentForValidation() {
     public void setAccountingDocumentForValidation(AccountingDocument accountingDocumentForValidation) {
         this.accountingDocumentForValidation = accountingDocumentForValidation;
     }
+    
+    protected CuDisbursementVoucherTaxService getCuDisbursementVoucherTaxService() {
+        if (cuDisbursementVoucherTaxService == null) {

```

Comment:
Since this class can use injection, that should be configured and this idiom would be unnecessary.